uploader/api_commands.py

```diff
@@ -59,6 +60,23 @@ def upload_video():
     file.save(os.path.join(current_unprocessed_folder, filename_with_timestamp))
     file.close()
 
+    # Upload to GCS bucket
+    bucket_name = 'ifpv-videos'
+    source_file_name = os.path.join(current_unprocessed_folder, filename_with_timestamp)
+    destination_blob_name = f"unprocessed_videos/{filename_with_timestamp}"
+
+    storage_client = storage.Client()
+    bucket = storage_client.bucket(bucket_name)
+    blob = bucket.blob(destination_blob_name)
+
+    generation_match_precondition = 0
+
+    blob.upload_from_filename(source_file_name, if_generation_match=generation_match_precondition)
```

Comment:
podemos probar con esto ??? no se si sea posible, para evitar tener que guardar el archivo y volverlo a cargar.
```suggestion
    blob.upload_from_file(file)
```

Reply:
No estoy seguro se debamos quitar la funcionalidad de NFS.

Reply:
Crearemos otro PR para esto. Cuando quitemos el NFS, se puede subir el video a Cloud Storage desde memoria.